examples/e2e/azure/resource_cluster.tf

```diff
@@ -45,6 +48,7 @@ resource "spectrocloud_cluster_azure" "cluster" {
   }
 
   machine_pool {
+    is_system_node_pool     = true
```

Comment:
Let's add corresponding validation for this rule and unit test too.

Reply:
is_system_node_pool is a bool type right? I can add a Default value.
What should I vaalidate in the schema?

Reply:
No not in the schema, there should be a check in the code that at least one pool has system nodes.

Reply:
As discussed, This will be taken care in ticket https://spectrocloud.atlassian.net/browse/PLT-294